.github/workflows/deploy-review-command.yml

```diff
@@ -63,6 +63,31 @@ jobs:
               "GITHUB_TRIGGER": "${{ github.event.client_payload.github.payload.comment.html_url }}"
             }
 
+      - name: Check status
+        id: status
+        if: success() || failure()
+        run: |
+          GITHUB_WORKFLOW_URL=$GITHUB_SERVER_URL/$GITHUB_REPOSITORY/actions/runs/$GITHUB_RUN_ID
+          GITLAB_PIPELINE_URL=$(curl --silent --header "PRIVATE-TOKEN: ${{ secrets.DEPLOY_ACCESS_TOKEN }}" "https://${{ secrets.DEPLOY_HOST }}/api/v4/projects/${{ secrets.DEPLOY_PROJECT_ID }}/pipelines/latest" | jq -r '.web_url')
+          ENVIRONMENT_URL=https://chat-${{ github.event.client_payload.pull_request.head.repo.full_name }}-pr-${{ github.event.client_payload.pull_request.number }}.${{ vars.DEPLOY_BASE_DOMAIN }}
```

Comment:
Taking the URL from GitLab API environment would be the best option . It may be necessary to add an environment for starting jobs in Gitlab.